### ISSUE TITLE
no download or other controls evident?

### ISSUE THREAD

**Issue by kberry** [houtianze/libby-audiobook-exporter-browser-extension#14]
Hi - thank you for the extension and the instructions for installing it. I loaded it in chrome (x86_64-linux, version 123.0.something, "(Official Build)") by adding --load-extension=/tmp/libby-audiobook-exporter-browser-extension-master to the command line, since the "load unpacked" button in chrome://extensions (which was indeed visible with developer mode on), did nothing at all. And now I see "Libby Audiobook Exporter 2.4" in the extensions list, so I think that part worked.

But, then I navigate to the book on libbyapp.com, after logging in with my library card, etc. After clicking "open audiobook" on the book web page, I end up at a page with a url like https://libbyapp.com/open/loan/51540410/10056907. It looks like a typical audio player screen (screenshot attached). I see nothing that looks like "download mp3s" or anything else that might be created by your extension, but I'm also not sure what I should see. Help? Thanks much for any info ...

P.S. I also tried ￼the "Libby (Legacy) Audiobook Exporter" extension, installed from the chrome web store, but, same issue,  I see no controls as shown in the video on its extension page.
![s](https://github.com/houtianze/libby-audiobook-exporter-browser-extension/assets/362855/86575bb2-b5f3-44bf-a19d-d50e3031aef4)

**Comment by mkgtu** [comment]
With the 2.4 extension from GitHub, look in the drop-down list of extensions from the browser toolbar (not the manage extensions page). Find Libby Audiobook Exporter I the list and use the little "pin" button to pin it to the toolbar so it's easy to access. 
  Then, when you open the audiobook, after it opens, refresh the page by using the normal refresh button (little circular arrow at left end of toolbar in Chrome). Now, after that refresh, click the Libby Audiobook Exporter icon/button that you pinned to the toolbar. That should open a small drop-down list with all the mp3 files. Tap the EXPORT button at the top of that list to start the downloads. (If refreshing the page doesn't work, refresh it again and maybe even again - third time's a charm)

TWO TIPS I used to, and still do occasionally, after refreshing the page, I also tapped the Play button to start playing the book for just a second or so, then stop it. It was just my way of making sure the page was actually loaded - by giving it a nudge. I don't know if it really helps, but it made me feel "in control" 🤣

TIP TWO  If you still can't get results, you can try what some are suggesting here has worked for them. Go to Site Settings and delete and reset all the Libby cookies under libbyapp.com. That will give you a clean slate. (Wouldn't hurt to delete/clear the browser cache while you're at it.) Then reopen the Libby website and reactivate it by entering your library cards. What has worked for others is re-entering the cards and PINS manually. But personally I'd be curious to know if activating/syncing your account by using a setup code from your Libby App would work just as well. It's certainly quicker and more complete, if using the setup code doesn't bring success with the extension, you can always repeat the process by doing it manually.

**Comment by kberry** [comment]
Now, after that refresh, click the Libby Audiobook Exporter
    icon/button that you pinned to the toolbar.

It worked!! I'm so happy :). Thank you so much!

I'm so grateful for your explicit descriptions of what to do in the UI.
(The GUI is completely non-intuitive to me.) Thank you again.

Maybe worth adding this description to your github docs.

A couple other unimportant notes:

    I also tapped the Play button to start playing the book for just a

I've tried that too :). It didn't make a difference in this case.

    personally I'd be curious to know if activating/syncing your account
    by using a setup code from your Libby App would work just as well.

I don't currently have the Libby app, so sorry, I can't satisfy your
curiousity :). I logged in to libbyapp.com in my desktop browser, which
worked fine. I did not have to log out of libbyapp.com and back in for
your process above to work.

Thanks again,
Karl
